spec/models/pull_request_spec.rb

```diff
@@ -30,12 +30,12 @@
   end
 
   describe ".needs_reposting" do
-    it "returns PRs that haven't been reviewed in over 30 minutes nor reposted" do
+    it "returns PRs that haven't been reviewed w/in the threshold nor reposted" do
```

Comment:
I'm open to suggestions about how to shorten this...otherwise, I'll just ignore it.